spec/integration/inventory_spec.rb

```diff
@@ -152,7 +152,7 @@ module Hawkular::Inventory::RSpec
         it 'Should list resources for feed' do
           resources = @client.list_resources_for_feed feed_id
 
-          expect(resources.size).to be(2)
+          expect(resources.size).to be >= 2
```

Comment:
Well, we must ask ourselves what are we really testing here, the fact that we should get feeds or that hawkular-services should have 2 (or 4, as the current configuration of the javaagent does).

If both, we can configure our hawkular-services's agent (as [done](https://github.com/josejulio/hawkular-client-ruby/blob/b4d5450fedd13ea055bc2a5f00ab752bed5d57ff/.travis/build_config_for_testing.rb) in this PR to increase its scan frequency) to be more specific on what we expect of it. 

That would be more fit on a separate PR on which we should decide what we really want to test and how to configure it.